tsl/test/sql/continuous_aggs.sql.in

```diff
@@ -1158,7 +1161,9 @@ FROM _timescaledb_catalog.chunk
 WHERE hypertable_id = :'MAT_HTID' and status = 1;
 
 --disable compression on cagg after decompressing all chunks--
+\set ON_ERROR_STOP 0
```

Comment:
Why this is needed? Didn't see any ERROR on output tests related to this change

Reply:
I randomly see below error when ALTER MATERIALIZED VIEW to disable compression is executed. So for safety i have set ON_ERROR_STOP.
+ERROR:  cannot change configuration on already compressed chunks
Anyways now i removed it.